ClLike/cl_like/limber.py

```diff
@@ -159,27 +165,33 @@ def _get_cl_data(self, cosmo, **pars):
                 ls = clm['l_eff']
             elif self.sample_bpw:
                 ls = self.l_sample
+            else:
+                raise RuntimeError("Something went wrong with the sampling!")
```

Comment:
Why do we need this now?

Reply:
It's a sanity check. If we have only prepared our code for two scenarios and none of them are used, it's because something went wrong.